styles.css

```diff
@@ -0,0 +1,621 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  list-style-type: none;
+  font-family: "Playfair Display", serif;
```

Comment:
- [ ] You should use font from the given design. Please use these fonts:

Fonts used originally: Cocogoose, Lato